awsutil/client_options.go

```diff
@@ -88,6 +81,31 @@ func (o *ClientOptions) Validate() error {
 	return nil
 }
 
+var configCache = make(map[string]*aws.Config)
```

Comment:
Nit: can you just put a comment here explaining why we do this for posterity?

Reply:
done!